mk/gotest.sh

```diff
@@ -47,7 +46,7 @@ elif [[ $# -eq 2 ]]; then
   if [[ ${TEST,,} == bench* ]]; then
     RUN=$RUN' -bench '$TEST
   fi
-  $SUDO GODEBUG=cgocheck=2 $DBG $GO test ./$TESTPKG -v -count=$TESTCOUNT $RUN
+  $SUDO GODEBUG=cgocheck=2 $DBG "$GO" test ./$TESTPKG -v -count=$TESTCOUNT $RUN
```

Comment:
is `$DBG` correct here? it's not set anywhere in this branch

Reply:
It should be deleted.